post_processing/contours_and_profiles.jl

```diff
@@ -69,15 +69,25 @@ function contour_plot!(
     end
     ts = getindex.(var_col_time_series, 1)
     zs = var_col_time_series[1][2]
+
+    #values_adjoint = hcat(getindex.(var_col_time_series, 3)...)
+    #values = permutedims(values_adjoint, (2, 1))
     values = hcat(getindex.(var_col_time_series, 3)...)'
+
     if isnothing(ref_var_col_time_series)
         limits = extrema(values)
     else
         ref_ts = getindex.(ref_var_col_time_series, 1)
         ref_zs = ref_var_col_time_series[1][2]
+
+        #ref_values_adjoint = hcat(getindex.(ref_var_col_time_series, 3)...)
```

Comment:
remove this?

Reply:
Same as the above. The main branch doesn't work for me :/ But the commented out lines do. Maybe we can keep them commented out just in case?